Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -1686,23 +1768,35 @@ namespace O3DE::ProjectManager
                 gemRepoInfo.m_path = gemRepoInfo.m_directoryLink = Py_To_String(repoPath);
 
                 QString lastUpdated = Py_To_String_Optional(data, "last_updated", "");
-                gemRepoInfo.m_lastUpdated = QDateTime::fromString(lastUpdated, RepoTimeFormat);
+                // this is NOT the same as RepoTimeFormat which uses 12 hour + am/pm
+                gemRepoInfo.m_lastUpdated = QDateTime::fromString(lastUpdated, "dd/MM/yyyy HH:mm");
```

Comment:
The date format of `dd/MM/yyyy` is confusing to US users.

Values such as 04/07/2023 would mean July 4th 2023.

Reply:
I agree! which is why I changed the display value in Project Manager to show the format used in the current locale so US users see `mm/dd/yyyy` and euro users see `dd/MM/yyyy`.
If I convert the underlying format to ISO then the only reason this could will remain is for backwards compatibility.